README.md

```diff
@@ -1,2 +1,69 @@
+
+[Australian Flammability Monitoring System](http://wenfo.org/afms/#_/Fenner%3AFMC%3AC6/gfe_fire_weather/_/_/_/4/8/2017/_)
+
+[Australian Environment Explorer](http://wenfo.org/ausenv/#/2016/Exposed_soil/Grid/Actual/States_and%20Territories/bar,options/-23.07/135.08/4/none/Roadmap/Opaque)
+
+
+### Input Data and Process
+---
+
+### Scripts
+---
+#### onetile.py
+Onetile is used to generate *one tile* of LFMC data for the [AFMS](http://wenfo.org/afms/#_/Fenner%3AFMC%3AC6/gfe_fire_weather/_/_/_/4/8/2017/_) system. The script uses a group of colour channels within satellite meta-data from [NASA's MODIS satellite](https://terra.nasa.gov/about/terra-instruments/modis) (sinusoidal projection) to calculate information about fuel moisture and reflectance on a specific tile of the world. Onetile is also capable of interpreting SPOT6 and SPOT7 satellite data and can be extended to support more. The world map on the sinusoidal projection grid can be found [here](https://modis-land.gsfc.nasa.gov/images/MODIS_sinusoidal_grid1.gif).
+
+The script takes a few command-line arguments to generate data however is more commonly used by launchmany.py - [see below](#launchmany.py)
+
+```
+-V, --version : returns the program version
+--year : a four digit year (20xx) to specify which dataset to process the tile on (defaults to current year)
+--tile : a specific sinusoidal tile (hXXvYY) on the world map to process (defaults to h31v10)
+--output_path : change the destination for tile output data to save to (defaults to /g/data/ub8/au/FMC/LVMC)
+```
+
+NOTE: To run onetile.py, you have to be in the *root* directory.
+
+
+#### launchmany.py
+Launchmany is used as a job queue for the NCI raijin supercomputer, used to generate a collection of tiles of LFMC data. The heart of the LFMC data generation belongs in onetile.py. This script takes a comma separated list of sinusoidal tiles to generate data for and schedules a set of jobs on raijin (using the raijin specific job handler) as generating many tiles over many years can take some time.
+
+For ease of use, launchmany has a few tile shortcuts builtin for countries around the world, such as Australia, Spain and South Africa. The script also calculates a walltime for each tile for logging purposes.
+
+```
+-V, --version : returns the program version
+--start_year : a four digit year (20xx) to start from to the current year, inclusive. (defaults to 2001)
+--tiles : a comma separated list of sinusoidal tiles (hXXvYY) on the world map to process, or a pre-defined shortcut. (defaults to shorcut australia)
+--output_path : change the destination for tile output data to save to (defaults to /g/data/ub8/au/FMC/LVMC)
+```
+
+#### modis.py
+This script is used as a general-purpose script for manipulating MODIS data, including a function to convert the sinusoidal grid coordinates (hXXvYY) to more conventional coordinates (x, y) and the get_reflectance function used by Onetile to calculate reflectance for the specific tile.
```

Comment:
"This script provides general-purpose functions for manipulating MODIS data, including loading reflectance and restoring physical coordinates to an array for a given tile."

Note: the x/y coords *are* sinusoidal, and reflectance is loaded rather than calculated.